### PR TITLE
Adding support for pageRequestTimeout

### PR DESCRIPTION
**Pull request by Brian-Courtois**
This PR responds to this issue:
 Add support for pageRequestTimeout #71 https://github.com/DevExpress/angular-testcafe/issues/71

### CODE REVIEW

**Comment by viktoria2506** [comment]
Thank you for the contribution to TestCafe. We will review it.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "angular-testcafe",
-  "version": "2.0.6",
+  "name": "angular-testcafe-sword",
```
Comment (miherlosev) [inline package.json:2]: Revert the `name` field.

File: package-lock.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "angular-testcafe",
+  "name": "angular-testcafe-sword",
```
Comment (miherlosev) [inline package-lock.json:2]: Please remove the `package-lock.json` file. This module will be used the latest version of packages.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/DevExpress/angular-testcafe.git"
```
Comment (miherlosev) [inline package.json:15]: Revert the `url` field.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "angular-testcafe",
-  "version": "2.0.6",
+  "name": "angular-testcafe-sword",
+  "version": "2.0.8",
```
Comment (miherlosev) [inline package.json:3]: `"version": "2.0.7",`

File: src/testcafe/schema.d.ts [hunk 5]
```diff
@@ -1,51 +1,52 @@
...
-    skipUncaughtErrors?: boolean;
-    speed?: number;
-    src: string | string[];
-    ssl?: string;
-    stopOnFirstFail?: boolean;
-    test?: string;
-    testGrep?: string;
-    testMeta?: Record<string, string>;
-    tsConfigPath?: string;
+  allowMultipleWindows?: boolean;
```
Comment (miherlosev) [inline src/testcafe/schema.d.ts:2]: Use the origin formatting here and below.

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Brian-Courtois) [reply package.json]: Done

File: package.json (hunk 3 above)
Comment (Brian-Courtois) [reply package.json]: Done

File: package.json (hunk 4 above)
Comment (Brian-Courtois) [reply package.json]: Done

File: src/testcafe/schema.d.ts (hunk 5 above)
Comment (Brian-Courtois) [reply src/testcafe/schema.d.ts]: Done, sorry for that. Forgot to disable prettier

**Comment by miherlosev** [comment]
Rebase please.

**Comment by Brian-Courtois** [comment]
> 
> 
> Rebase please.

Not sure I understand, what do you mean by rebase ? There is only one branch in my fork

**Review by miherlosev — APPROVED** [review]

**Review by AlexKamaev — APPROVED** [review]

**Pull request merged**